### ISSUE TITLE
Support for browsers that shipped with native Web Components

### ISSUE THREAD

**Issue by motss** [leaflet-extras/leaflet-map#75]
`leaflet-map` is surprisingly still depending on the web components polyfill. It should provide support for native Web Components that shipped on Chrome and Opera.

**Comment by nhnb** [comment]
Yes, the demo pages include the polyfill. But I am not aware of a dependency from any component files on the webcomponent polyfill.

**Comment by motss** [comment]
@nhnb I'm using `leaflet-map` on Chrome with native Shadow DOM and without loading the web components polyfill. It is just broken without the polyfill and works fine with the polyfill which is why I said that it depends on the polyfill and none of files has that dependency.
